tests/golos.ctrl_test_api.hpp

```diff
@@ -11,6 +14,13 @@ struct golos_ctrl_api: base_contract_api {
     const name _majority_name = N(witn.major);
     const name _supermajority_name = N(witn.smajor);
 
+    void initialize_contract(account_name token) {
```

Comment:
here (and in vesting/posting/referral) can be default value `token = cfg::token_name` to make most calls parameter-less

Reply:
You are right, this seems like boilerplate. But I prefer clarity in the fixture-constructor. Therefore, I tend to use mandatory arguments, as they clearly show dependencies.